Makefile

```diff
@@ -27,6 +27,10 @@ $(BIN)/%: | $(BIN) ; $(info $(M) building $(PACKAGE)…)
 	   rm -rf $$tmp ; exit $$ret
 
 KO = $(or ${KO_BIN},${KO_BIN},$(BIN)/ko)
+
+PIPELINES ?= latest
+TRIGGERS ?= latest
+
```

Comment:
can we do it for dashboard also ?

Reply:
I think it makes sense.
We may need to take care of Kubernetes and Openshift.

Reply:
yep @nikhil-thomas is working on adding results and dashboard as part of this :)